instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Comment from "../Comment/Comment";
+import PropTypes from "prop-types";
+import moment from "moment";
+
+const CommentSection = ({
+  comments,
+  timestamp,
+  newCommentText,
+  changeHandlerNested,
+  addNewComment
+}) => {
+  return (
+    <section className="comments">
+      {comments.map((comment, index) => (
+        // Used index for key for now - Date.now() wasn't working for some reason:
+        <Comment comment={comment} key={index} />
```

Comment:
I believe I already talked with you about how to optimize this, but bringing in the `uuid` library is a clean way of generating unique id's for what you're mapping out